### DISCUSSION TITLE
Why is Orca so much more laggy than Bambu on Mac?

### DISCUSSION THREAD

**Discussion by pennystockss** [OrcaSlicer/OrcaSlicer#2423] [General]
I've noticed on Mac, Orca is several times more laggy than Bambu. With small stuff on screen or not in slicing screen it's alright. But 5-10 full color objects and Orca is almost unusable after sliced. 2021 MacBook Pro. Anyone else having this happen? Never had this issue on Windows with an i7/3060 laptop.

https://github.com/SoftFever/OrcaSlicer/assets/60720749/55ac445f-b9db-4db6-8524-8637dc3ef530

**Comment by hotellonely** [comment]
It's the samething for M3 Max machines :(
